app/src/main/java/comcmput301f16t01/github/carrier/UserProfileActivity.java

```diff
@@ -108,10 +154,14 @@ public void editEmailAddress(View v){
         cancelButton.setVisibility(View.VISIBLE);
         EditText emailView = (EditText) findViewById(R.id.EmailEditText);
         // Set it so the user can edit the EditText
-        emailView.setFocusable(true);
+        emailView.setFocusableInTouchMode(true);
```

Comment:
I just looked this up. It's interesting, what "Focus" and "FocusInTouchMode" is.